earn/src/components/borrow/modal/BorrowModal.tsx

```diff
@@ -31,7 +195,74 @@ export default function BorrowModal(props: BorrowModalProps) {
       }}
       maxHeight='650px'
     >
-      <div className='w-full'></div>
+      <div className='flex flex-col items-center justify-center gap-8 w-full mt-2'>
+        <div className='flex flex-col gap-1 w-full'>
+          <Text size='M' weight='bold'>
+            Borrow Amount
+          </Text>
+          <TokenAmountSelectInput
+            inputValue={borrowAmount}
+            onChange={(value) => {
+              const output = formatNumberInput(value);
+              if (output != null) {
+                const truncatedOutput = truncateDecimals(output, borrowToken.decimals);
+                setBorrowAmount(truncatedOutput);
+              }
+            }}
+            onSelect={(option: Token) => {
+              setBorrowAmount('');
+              setBorrowToken(option);
+            }}
+            options={tokenOptions}
+            selectedOption={borrowToken}
+          />
+        </div>
+        <div className='flex flex-col gap-1 w-full'>
+          <Text size='M' weight='bold'>
+            Summary
+          </Text>
+          <Text size='XS' color={SECONDARY_COLOR} className='overflow-hidden text-ellipsis'>
+            You're borrowing{' '}
+            <strong>
+              {borrowAmount || '0.00'} {borrowToken.ticker}
+            </strong>{' '}
+            using this{' '}
+            <strong>
+              {marginAccount.token0.ticker}/{marginAccount.token1.ticker}
+            </strong>{' '}
+            smart wallet. Your total liabilities for this token in this smart wallet will be{' '}
```

Comment:
Instead of the textual summary here, it may be more useful to show a hypothetical health bar

Reply:
I'll see what I can do. With that said, I think it may be better to include both, so the user knows about both what is being transacted, and what the impact on their health is.